docker-compose-lib.yaml

```diff
@@ -218,6 +221,9 @@ services:
       - SQLALCHEMY_POOL_RECYCLE
       - SQLALCHEMY_POOL_SIZE
       - SQLALCHEMY_MAX_OVERFLOW
+      - SQLALCHEMY_SLAVE_POOL_RECYCLE
+      - SQLALCHEMY_SLAVE_POOL_SIZE
+      - SQLALCHEMY_SLAVE_MAX_OVERFLOW
```

Comment:
Same

Reply:
@sbrunner : Yes we have default values in Dockerfile, but I would like to set them in docker-compose too, using env files.

We may have different values in demo and production projects.